src/openms/include/OpenMS/ANALYSIS/SEQUENCE/NeedlemanWunsch.h

```diff
@@ -0,0 +1,39 @@
+#include <vector>
+#include <OpenMS/DATASTRUCTURES/String.h>
+#include <OpenMS/OpenMSConfig.h>
+
+
+namespace OpenMS
+{
+  class OPENMS_DLLAPI NeedlemanWunsch
+  {
+
+  public:
+    enum class ScoringMatrix
+    {
+      PAM30MSMatrix,
+      identityMatrix
+    };
+
+    NeedlemanWunsch(ScoringMatrix matrix, int penalty);
+
+    ~NeedlemanWunsch()=default;
+
+    double align_(const String& seq1, const String& seq2);
```

Comment:
for the next three functions, remove the trailing `_` to obey naming conventions